### DISCUSSION TITLE
Public access not working

### DISCUSSION THREAD

**Discussion by wsherwin01** [community/community#112293] [Repositories]
### Select Topic Area

General

### Body

13 Mar 2024.  I have set up a GitHub repository https://github.com/wsherwin01/DnaDot with licence GNU General Public License v3.0.  For some reason, other users can see the repository, but are not allowed to download.  How do I fix this please?  Bill Sherwin W.Sherwin@unsw.edu.au

**Comment by DeandrePetersen** [comment]
Go to your GitHub repository's settings by clicking on the "Settings" tab. Under the "Options" section, ensure that the "Features" settings, such as issues, projects, and wiki, are enabled as desired. In the "Options" section of your repository settings, under "Danger Zone," there's an option for "Repository visibility." Ensure that it is set to either "Public" or "Private" based on your preference. If it's set to "Private," only users with access to the repository will be able to download its contents. Make sure that the users who need to download the repository have the appropriate permissions. If your repository is public, anyone should be able to download it. If it's private, ensure that the users have been added as collaborators with at least "Read" access.

**Reply by wsherwin01** [reply]
Thansk, But in Danger Zone, it says "This repository is currently public."  BUt several users have said that it is not.
